types/oscal/profile/profile.go

```diff
@@ -33,15 +33,15 @@ type Merge struct {
 // A Custom element frames a structure for embedding represented controls in
 // resolution.
 type Custom struct {
-	IdSelectors      []Call  `xml:"call,omitempty" json:"calls,omitempty"`
```

Comment:
Is this reverted by intention? the profiles at hand have `calls` attributes and not `idSelectors`

Reply:
This was re-generated from the profile metaschema which, as we've discussed, has a bug per https://github.com/usnistgov/OSCAL/issues/287. I'll revert these changes.